### ISSUE TITLE
Error: java.io.IOException: write failed: ENOSPC (No space left on device)

### ISSUE THREAD

**Issue by loremattei** [wordpress-mobile/WordPress-Android#13761]
**Sentry Url**: https://sentry.io/share/issue/4c3e38a670114daaa5b02dc692119383/
**User Count**: 97
**Count**: 296
**First Release**: org.wordpress.android-14.6.1
**First Seen**: 2020-04-28T23:31:58Z
**Last Seen**: 2021-01-13T14:37:49Z
**24 Hours**: 10
**30 Days**: 190

```
Error: java.io.IOException: write failed: ENOSPC (No space left on device)
     at org.wordpress.android.util.helpers.logfile.LogFileWriter$write$1.run(LogFileWriter.kt:41)
...
(3 additional frame(s) were not displayed)
```

**Comment by khaykov** [comment]
@jkmassel I don't have all the context, but it looks like this issue started to happen after introduction of a Log Persistence in this [commit](https://github.com/wordpress-mobile/WordPress-Utils-Android/commit/f65fd7841420db663329de536f420575c5d0b16b#diff-5104b4aeb4400e0c0f9a5fb7751074c4eff44b13dd777ddffb81ac0cb949705d). Maybe you know what might be the case here?

**Comment by designsimply** [comment]
**Events in the last 90d:** 367
**Users affected in the last 90d:** 106
**Majority of cases are:** Android 5.x or Android 6.x
<sub>WORDPRESS-ANDROID-B2K: https://sentry.io/share/issue/4c3e38a670114daaa5b02dc692119383/</sub>

<a href="https://sentry.io/organizations/a8c/discover/results/?display=daily&field=timestamp&field=version&field=os&field=device.family&field=culprit&name=All+Events&project=1438088&query=issue.id%3A1636269460&sort=-timestamp&statsPeriod=90d&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1&yAxis=count%28%29">![image](https://user-images.githubusercontent.com/1119271/105563841-cf92d880-5cdc-11eb-804e-83ab17181eee.png)</a>

**Comment by renanferrari** [comment]
Closing this as it seems to be a duplicate of #13271 which was fixed by #14103 and https://github.com/wordpress-mobile/WordPress-Utils-Android/pull/62.
